DoubleEngine.xcodeproj/xcshareddata/xcschemes/DoubleEngine.xcscheme

```diff
@@ -31,7 +31,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```

Comment:
Can we revert these changes, please?

Reply:
it's intended, since the purpose is to check performance. 

Run is used to check the simple memory counter in Xcode, Profile is for memory in instruments.

Reply:
Ahh, I just use Profile, did you notice a difference between Release and Profile?

Reply:
There's the vm-service isolate in profile. What do you do in the Podfile?